callosum/exceptions.py

```diff
@@ -6,6 +6,17 @@ class ClientError(CallosumError):
     pass
 
 
+class ParamError(ClientError):
```

Comment:
You have used this error in a binder class, which means it's server-side, but `ParamError` is subclassing `ClientError`.
Since `ParamError` is a very generic error that represents any type of invalid configuration/combination of parameters, we could make it a direct subclass of `CallosumError` and possibly `ValueError` as well.